src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -34,6 +32,49 @@
        (map keyword)
        set))
 
+(defn- get-questions-for-page [working-attributes library-schema page-num]
+  (when (< page-num (count (:wizard library-schema)))
+    (let [page-props (get-in library-schema [:wizard page-num])
+          {:keys [questions enumerate optionSource options]} page-props]
+      (if optionSource
+        (let [option-value (get working-attributes (keyword optionSource))]
+          (when-let [option-match (some->> option-value keyword (get options))]
+            (map option-match [:questions :enumerate])))
+        [questions enumerate]))))
+
+(defn- convert-empty-strings [attributes]
+  (utils/map-values
+   (fn [val]
+     (if (or (coll? val) (string? val))
+       (not-empty val)
+       val)) attributes))
+
+(defn- render-summary-page [attributes library-schema invalid-attributes]
+  [:div {}
+   ;; TODO (as part of 1321?) don't let people publish if you have chosen skip
+   ;(if (= (:library:useLimitationOption attributes) "skip") ;; do this is in a non-hardcoded way
+   ;  "Note, you cannot publish this dataset until you define the Data Use Limitations.")
+
+   (if (not-empty invalid-attributes)
+     [:div {:style {:fontSize "14px" :color (:exception-state style/colors)}}
+      "Please fill in missing attributes before saving."
+      [:ul {:style {:fontSize "13px"}}
+       (map (fn [attribute]
+              [:li {} (get-in library-schema [:properties (keyword attribute) :title])])
+            invalid-attributes)]])
+
+   (style/create-paragraph
+    [:div {}
+     (let [questions (first (get-questions-for-page (convert-empty-strings attributes) library-schema 0))]
+       (map (fn [attribute]
+              (if (not-empty (str (attributes (keyword attribute))))
+                (library-utils/render-property library-schema attributes (keyword attribute))))
+            questions))
+     (if (= (:library:useLimitationOption attributes) "orsp") ;; TODO: change this so not hardcoded
+       (library-utils/render-property library-schema attributes :library:orsp)
+       (library-utils/render-consent-codes library-schema attributes))])])
```

Comment:
I might just make a new ticket for it, I mentioned it to Matt and he said it would involve more changes to the json schema and I'd kinda like to get this PR through so we can really work on discover